cmd/oci-image-validate/main.go

```diff
@@ -145,6 +145,10 @@ func (v *validateCmd) Run(cmd *cobra.Command, args []string) {
 		exitcode = 1
 	}
 
+	if exitcode == 0 {
+		v.stdout.Println("validation succeeded")
```

Comment:
OK is the determination of each arg, rather than the overall validate decision.
I would like to show that the entire order of judgment (args), rather than a single decision (arg).